Hitchhiker galaxy design

codingunicorn opened the pull request:
This is the first change for the Galaxy theme including the background-image. Not connected to a raised issue.

lukaselmer commented on data/initial-data.json:
```diff
@@ -2,7 +2,13 @@
-      "logos": []
+      "logos": [
+        {
+          "logoUrl": "../images/logos/ERNI.png",
```
I suggest to use lower case for filenames 😉

lukaselmer replied:
Regarding file size: this is super small, nice 😊

lukaselmer commented on docs/tutorials/configure-app.md:
```diff
@@ -59,12 +59,12 @@ window.ENV = 'dev';
       // data only applicable for the home page
       "video": {
-        "title": "GDG DevFest Ukraine 2016",
+        "title": "GDG DevFest Ukraine 201",
```
Mm we don't need to change this file, right? Can we revert these changes?

lukaselmer commented on src/styles/shared-styles.html:
```diff
@@ -11,9 +11,12 @@
+        --dark-primary-color:       #032d54;
+       /*  --default-primary-color:    #21a4c4; */
+        --default-primary-color:   #032d54; 
+        --cta-color:                #80CBC4;
```
What is this variable used for?

lukaselmer commented on src/styles/shared-styles.html:
```diff
@@ -11,9 +11,12 @@
+       /*  --default-primary-color:    #21a4c4; */
+        --default-primary-color:   #032d54; 
+        --cta-color:                #80CBC4;
+        /* --light-primary-color:      #66d6f7; */
```
Let's delete the comments - no dead code 😉

lukaselmer commented on docs/tutorials/configure-app.md:
```diff
@@ -59,12 +59,12 @@ window.ENV = 'dev';
     "headerSettings": {
       "backgroundColor": "#FF9800",
-      "backgroundImage": "/images/backgrounds/home.png",
+      "backgroundImage": "/images/backgrounds/galaxy-bg.png",
```
The image looks very nice 😊 two things:

* Did you check that it looks good with different screen resolutions?
* The file is nearly 1MB in size, which is quite large. Can you minify it without visible quality loss?

codingunicorn replied:
a) Yes, but I'd want to discuss with you largening the header image anyways, as this is now is causing responsive issues on some devices. 
b) I already minified it to the max.

codingunicorn commented:
Hey @lukaselmer thanks for your review. I must have overlooked some things in the rush, which I fixed now. I also changed the CTA Color again to the Turquoise from the Style guidelines (see Screenshot below). 


![galaxy-theme](https://user-images.githubusercontent.com/19932169/43361727-f733847e-92d7-11e8-8c90-b1b00c5ffaf0.PNG)

Additionally, I added comments to your Background image review. Let me know if this is okay or if you'd want to further minify the image.

ChristofBuechi commented:
@lukaselmer Is there still some action todo here ? I think the design is merged ? @codingunicorn is waiting for a feedback for this merge request. Whats the status ?
